backend/compact-connect/lambdas/python/data-events/handlers/encumbrance_events.py

```
@@ -59,11 +86,540 @@ def license_encumbrance_lifted_listener(message: dict):
+                        license_type_abbreviation=license_type_abbreviation,
+                        effective_lift_date=effective_lift_date,
+                        event_batch_writer=event_batch_writer,
+                    )
+
+        logger.info(
+            'Successfully processed license encumbrance lifting event', privileges_unencumbered=len(affected_privileges)
+        )
+
+
+@sqs_handler
+def privilege_encumbrance_listener(message: dict):
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)